_posts/2022-09-12-web-schema-sandbox.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: news_item
+title: "Web Schema Sandbox"
+date: 2022-09-12
+categories: news ion-schema
+---
+
+A Web Schema Sandbox was added to this public GitHub website on 12 September 2022.
+
+This sandbox can be used to validate Ion values for a particular type defined in Ion Schema. 
+The sandbox is created using `ion-schema-rust` latest pre-release version.
```

Comment:
I thought it used `0.4.0`, but `ion-schema-rust 0.5.0` was just released. Is that correct?

Reply:
Yes, but it does use `0.5.0 `as the wasm that was generated used `ion-schem-rust 0.5.0`

Reply:
Cool!